packages/flutter_tools/lib/src/web/file_generators/flutter_js.dart

```diff
@@ -96,7 +97,7 @@ _flutter.loader = null;
     _waitForServiceWorkerActivation(serviceWorker, entrypointUrl) {
       if (!serviceWorker || serviceWorker.state == "activated") {
         if (!serviceWorker) {
-          console.warn("Cannot activate a null service worker. Falling back to plain <script> tag.");
+          console.warn("Cannot activate a null service worker.");
```

Comment:
Not in scope for this PR, but is it possible to put this JS code in a separate `.js` file then read it from there? It would be significantly better to look at this code with proper syntax highlighting. The IDE would also be able to catch obvious syntax errors, etc.

Reply:
(PS: Feel free to add into that issue, it's for brainstorming right now)